network/dime_message.ksy

```diff
@@ -0,0 +1,115 @@
+meta:
+  id: dime_message
+  title: DIME (Direct Internet Message Encapsulation) Message
+  file-extension:
+    - dim
+    - dime
+  xref:
+    mime: application/dime
+    wikidata: Q1227457
+  license: CC0-1.0
+  bit-endian: be
+  endian: be
+  encoding: ASCII
+doc: |
+  Direct Internet Message Encapsulation (DIME)
+  is an old Microsoft specification for sending and receiving
+  SOAP messages along with additional attachments,
+  like binary files, XML fragments, and even other
+  SOAP messages, using standard transport protocols like HTTP.
+  
+  Sample file: `curl -L
+  https://github.com/kaitai-io/kaitai_struct_formats/files/5890499/scanner.dump.gz
```

Comment:
If the sample file you referenced in https://github.com/kaitai-io/kaitai_struct_formats/pull/409#discussion_r566909282 is better for testing, perhaps this one can be replaced?

```suggestion
  https://github.com/kaitai-io/kaitai_struct_formats/files/5894723/scanner_withoptions.dump.gz
```

Reply:
It can and it should.